uw-frame-components/static.html

```diff
@@ -9,8 +9,8 @@
   }
 
   .slowfade {
```

Comment:
sloooow fade. I don't think we want to make this longer to try to solve the pre-css problem. (Or at least we should use a different easing than linear: http://easings.net/ ) This slow fade is the reason for that period of unresponsiveness on page load that I've heard @apetro grumble about :)